app/src/main/java/ch/itomy/xendit_example/CreateTokenActivity.java

```diff
@@ -102,11 +106,11 @@ public void onError(XenditError xenditError) {
         };
 
         if (isMultipleUse) {
-            xendit.createMultipleUseToken(card, callback);
+            xendit.createMultipleUseToken(card, shouldAuthenticate, callback);
```

Comment:
We shouldn't need to do it for multiple use. They can just not call createAuthentication